src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -436,12 +436,7 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                     ->info('validation configuration')
                     ->canBeEnabled()
                     ->children()
-                        ->scalarNode('cache')
-                            ->beforeNormalization()
-                                // Can be removed in 3.0, once ApcCache support is dropped
-                                ->ifString()->then(function ($v) { return 'apc' === $v ? 'validator.mapping.cache.apc' : $v; })
```

Comment:
@jderusse I don't agree. There is no BC break in 3.0 if we remove the apc logic. Configuring apc in 3.0 does not work anyway. `ApcCache` does not exist anymore.

So we should go with option 1 as planned: Add missing deprecation in 2.8 and remove `validator.mapping.cache.apc` service in 3.0